experiments/AtmosLES/dycoms.jl

```diff
@@ -395,7 +395,7 @@ function config_dycoms(
 end
 
 function config_diagnostics(driver_config)
-    interval = "1000steps"
+    interval = "10000steps"
```

Comment:
I see this sort of interval specification in quite a few experiments and it is problematic: change the poly order, the time stepper, the CFL, or any number of other things and the number of timesteps changes and you have to remember to change this also (and it's is easy to miss). I suggest passing `timeend` to this function and using something like:
```julia
ssecs = cld(timeend, 2) + 10
interval = "$(ssecs)ssecs"
```
Which should produce diagnostics output at `t = 0`, `t ~= timeend/2` and `t=timeend`.

Reply:
Yikes, I did not know this. Will update.

Reply:
Thanks!

Reply:
Thanks! This is a great tip. I will update my other setups too.